books/free-programming-books-es.md

```diff
@@ -95,6 +97,11 @@
 * [Teoría de la Computación: Lenguajes, Autómatas, Gramáticas](http://ciencias.bogota.unal.edu.co/fileadmin/Facultad_de_Ciencias/Publicaciones/Archivos_Libros/Libros_Matematicas/_Teoria_de_la_Computacion___lenguajes__automatas__gramaticas._/teoriacomputacion.pdf) - Rodrigo De Castro Korgi (PDF)
 
 
+#### Desarrollo Web
+
+* [Diferencia entre Frontend y Backend](https://programacionymas.com/frontend-vs-backend) - Blog de "Programación y más" (HTML)
```

Comment:
Otro Blog Post?

Reply:
Not part of the blog. Shared as a guide.
We have similar guides in the list.
But again, it's up to you. Just let me know please.

Reply:
Please, should be removed